### PR TITLE
add W3C Web-App Manifest w/ icons

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,6 +8,8 @@
     <script src="build/build.js"></script>
     <script src="vendor/aframe-bmfont-component.min.js"></script>
     <script src="assets/data/waves.js"></script>
+    <link rel="manifest" href="manifest.webmanifest">
```
Comment (cvan) [inline index.html]: lmk if I should put this in the `assets/` directory

**Comment by fernandojsg** [comment]
@cvan sorry I totally missed this one. @feiss if you agree with the icons I'll just merge it

**Comment by feiss** [comment]
Thanks Chris!

Mmm.. I'm not sure about the image. What about this? (can you generate the other sizes automagically?)

icon

**Comment by cvan** [comment]
@feiss thanks! can you make a version of the face that's not cropped? ideally, if you can make the background transparent, that'd be good too. especially on nice Android mobile phones (and on Chrome desktop), high-res icons with transparent backgrounds stand out and look much, much better.

**Comment by feiss** [comment]
Yeah, here you are!

icon

**Comment by cvan** [comment]
perfect, this looks great! I shall update the PR and re-generate the icons. thanks a ton, @feiss! 👍

**Comment by fernandojsg** [comment]
@cvan any updates on this?

**Comment by cvan** [comment]
@fernandojsg cleaned this up. it's ready to go 👍

**Comment by cvan** [comment]
I deleted my repo, so I'll have to re-open this as a separate, new PR

**Pull request closed without merging**